aws/lambda/log-classifier/src/bedrock.rs

```diff
@@ -32,7 +35,7 @@ use prompts::FIND_ERROR_LINE_PROMPT;
 /// 2. It searches for this extracted content in the log.
 ///
 /// If both steps succeed, it returns the matching log line. Otherwise, it returns None.
```

Comment:
Would be nice to update the comment here, that if succeeds it returns log line and its index